src/routes/DigitalAssets/routes/AddAsset/modules/addAsset.js

```diff
@@ -0,0 +1,191 @@
+// @flow
+
+export const OPEN_VIEW = '@@addAsset/OPEN_VIEW'
+export const CLOSE_VIEW = '@@addAsset/CLOSE_VIEW'
+
+export const SET_FIELD = '@@addAsset/SET_FIELD'
+export const SET_INVALID_FIELD = '@@addAsset/SET_INVALID_FIELD'
+
+export const START_ASSET_LOADING = '@@addAsset/START_ASSET_LOADING'
```

Comment:
More obviously to name constants & actions SUBMIT_REQUEST/SUCCESS/ERROR
We really should try to unify our modules.